core/worker/src/main/java/com/cognifide/aet/worker/drivers/chrome/ChromeCommunicationWrapperImpl.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.worker.drivers.chrome;
+
+import com.cognifide.aet.job.api.collector.HttpRequestExecutor;
+import com.cognifide.aet.job.api.collector.JsErrorLog;
+import com.cognifide.aet.job.api.collector.ProxyServerWrapper;
+import com.cognifide.aet.job.api.collector.WebCommunicationWrapper;
+import com.google.common.collect.Ordering;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.function.Predicate;
+import java.util.logging.Level;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.logging.LogEntries;
+import org.openqa.selenium.logging.LogEntry;
+import org.openqa.selenium.logging.LogType;
+
+public class ChromeCommunicationWrapperImpl implements WebCommunicationWrapper {
+
+  private static final LogEntryToJsError LOG_ENTRY_TO_JS_ERROR = new LogEntryToJsError();
```

Comment:
Lets change from

```
private static final LogEntryToJsError
```
to

```
private static final Function<LogEntry, JsErrorLog>
```
to use interface in declaration so that it is obvious that `LOG_ENTRY_TO_JS_ERROR` is a function.

Reply:
ok